src/main/java/com/ericsson/ei/notifications/EmailSender.java

```diff
@@ -96,12 +96,10 @@ public MimeMessage prepareEmailMessage(String recipients, String mapNotification
      * at application.proporties are empty.
      * */
     private void setDefaultValues() {
-        if (sender.isEmpty() && subject.isEmpty()) {
-            sender = "noreply@ericsson.com";
-            subject = "Email Subscription Notification";
-        } else if (!sender.isEmpty() && subject.isEmpty()) {
+       if (subject.isEmpty()) {
```

Comment:
Much beter! I wasn't aware that Spring interpreted it as empty values when the property is declared but no value. But this approach is good.